app/src/main/java/co/touchlab/kampkit/android/BreedViewModel.kt

```diff
@@ -32,6 +34,7 @@ class BreedViewModel : ViewModel(), KoinComponent {
         observeBreeds()
     }
 
+    @FlowPreview
```

Comment:
Do `@UseExperimental(FlowPreview::class)` here and then you don't need to annotate callers

Reply:
Oh, cool, I'll do that! I didn't know there was any difference between the 2 annotations.